test/acceptance/test/add_tests.go

```diff
@@ -962,11 +959,17 @@ var _ = Describe("Weave GitOps Add Tests", func() {
 
 		By("And I should not see wego components in the remote git repo", func() {
 			pullGitRepo(repoAbsolutePath)
-			folderOutput, _ := runCommandAndReturnOutput(fmt.Sprintf("cd %s && ls -al", repoAbsolutePath))
+			folderOutput, _ := runCommandAndReturnStringOutput(fmt.Sprintf("cd %s && ls -al", repoAbsolutePath))
 			Expect(folderOutput).ShouldNot(ContainSubstring(".wego"))
 			Expect(folderOutput).ShouldNot(ContainSubstring("apps"))
 			Expect(folderOutput).ShouldNot(ContainSubstring("targets"))
 		})
+
+		By("And I can see wego app status", func() {
+			statusOutput := runCommandAndReturnSessionOutput(fmt.Sprintf("%s app status %s", WEGO_BIN_PATH, appName))
+			Eventually(statusOutput).Should(gbytes.Say(
+				`Latest successful deployment time: 202[0-9]-[0-9][0-9]-[0-9][0-9]T[0-9][0-9]:[0-9][0-9]:[0-9][0-9]Z\n.*\n.*gitrepository\/` + appName + `\s*True\s*Fetched revision: main\/.{40}\smain\/.{40}\s*False\s*\n.*\n.*kustomization\/` + appName + `\s*True\s*Applied revision: main\/.{40}\smain\/.{40}\s*False`))
```

Comment:
Nit: For the readability of code, I would expand it to multiple lines.

Reply:
Made requested changes